src/app/application/components/app-collection/AppCollection.vue

```diff
@@ -84,11 +96,28 @@ const click = (e: MouseEvent) => {
     }
   }
 }
+
+function getCellAttributes({ headerKey }: any) {
+  return {
+    class: {
+      [`${headerKey}-column`]: true,
+    },
+  }
+}
```

Comment:
I would rather keep the `class` inline please.

Reply:
Why? Right now, we keep moving more and more logic into templates where it doesn’t belong. It’s already hard enough to parse what’s going on with a `KTable` that is fed a bunch of props, but adding inline functions to that makes it even worse.

Reply:
AppCollection.vue is 94 lines long. The template/html in there is 36 lines long, the vast majority of those 36 lines are just setting an attribute.

How hard is that to parse? Harder to parse than our listing components used to be?

Reply:
Templates should have hardly any logic in them at all. I’m just putting this function where it belongs.

If there is nothing particularly wrong with this, I’m leaving this as is.

Reply:
Where is the logic?

Reply:
There isn’t really a point arguing this. Tell if this is a blocker for you and I’ll move it back.

Reply:
This time around its a blocker for me.

The reasons you have given for doing this:

1. It's hard to follow. Its not.
2. It's logic and therefore shouldn't be in the template. There's no logic.
3. There's nothing wrong with it. There was nothing wrong with it in the first place and no reason to change (see points 1 and 2)

Happy to hear more opinion, but as of right now, this is a pointless conversation, I think we can agree on that?

Happy to revisit once we have this stream of work complete. The code we are talking about is in a centralized place and not spread out in multiple places in the application.

Reply:
It’s not useful to argue this. I’ve moved it back.